.github/workflows/ci.yml

```diff
@@ -0,0 +1,69 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request: ~
+
+jobs:
+  # derived from https://dev.to/felipetofoli/github-actions-for-net-full-framework-build-and-test-299h
+  NET_Tests:
+    runs-on: windows-2019
+    strategy:
+      matrix:
+        dotnet-version: [ '35', '40' ]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup variables
+        id: test_file
+        run: echo "::set-output name=test_file::EasyPost.Tests.Net${{ matrix.dotnet-version }}"
+      - name: Setup MSBuild
+        if: always()
```

Comment:
Do we need the `if: always()` bit here? This should automatically run anyway and shouldn't fail right? We trust Microsoft's action to work every time?

Reply:
No, this is a hold-over from a previous version of the script. Will remove.

Reply:
This is to ensure that, i.e. the 4.0 runner continues to run even if the 3.5 runner fails for whatever reason.

Reply:
To keep our build hours low, we may not want to run all jobs if one fails because we know we'll need to make changes anyway to get it working. Always running every job could eventually become costly. Thoughts?

Reply:
I removed it and, as expected, it halted later tests when one failed. My concern is, the other tests didn't get a chance to run, so we wouldn't know if just one test failed or they all failed because of something we did?
Also, sometimes the failure isn't necessarily something wrong with our code. One time, the CI failed for a platform simply because downloading the NuGet packages to set up the test stalled.